iree/compiler/Conversion/LinalgToLLVMGPU/Common.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 Nod Labs
+// Copyright 2021 Google LLC
```

Comment:
Can we pick a more specific file name than Common.cpp? Maybe ConvertToLLVMCommon.cpp or just ConverToLLVM.cpp?

Reply:
Sure I will rename it ConvertToLLVM.cpp